### DISCUSSION TITLE
What should i do with the domain

### DISCUSSION THREAD

**Discussion by LankyBox01** [theforumhelpers/theforumhelpers.github.io#147] [Show and tell]
Seriously. I don't know how long i've been holding this domain.
https://theforumhelpers.glitch.me/

I got so bored that i made my own thing:
https://theforumhelpers.glitch.me/data.html

**Comment by leahcimto** [comment]
Well, that's up to you. What would _you_ like to do with it?

**Reply by LankyBox01** [reply]
I was originally going to do a post analyser for the scratch forums using scratchdb, but i realized scratchdb is really slow when getting posts.

**Reply by FunctionalMetatable** [reply]
@CodeGuy92 https://scratchdb.lefty.one/v3/forum/post/info/:post-id

**Reply by LankyBox01** [reply]
I know about v3, but how do i get all of the post IDs?

**Reply by Accio1** [reply]
Since this conversation seems to be no longer related to this site, please move it elsewhere.

**Reply by LankyBox01** [reply]
You can delete it if u want
